test/btest.cfg

```diff
@@ -1,5 +1,5 @@
 [btest]
-TestDirs    = procs
+TestDirs    = procs cmd
```

Comment:
Nit: Maybe keep these sorted.

Reply:
Separately, I think format/ or output/ or flag/f/ would be a better name for the test directory.